pennylane_lightning/lightning_gpu/lightning_gpu.py

```diff
@@ -0,0 +1,755 @@
+# Copyright 2022-2023 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+r"""
+This module contains the :class:`~.LightningGPU` class, a PennyLane simulator device that
+interfaces with the NVIDIA cuQuantum cuStateVec simulator library for GPU-enabled calculations.
+"""
+
+from warnings import warn
+import numpy as np
+
+from pennylane_lightning.core.lightning_base import (
+    LightningBase,
+    LightningBaseFallBack,
+)
+
+try:
+    from pennylane_lightning.lightning_gpu_ops import (
+        allocate_aligned_array,
+        backend_info,
+        best_alignment,
+        get_alignment,
+        StateVectorC128,
+        StateVectorC64,
+        MeasurementsC128,
+        MeasurementsC64,
+        is_gpu_supported,
+        get_gpu_arch,
+        DevPool,
+    )
+
+    from ctypes.util import find_library
+    from importlib import util as imp_util
+
+    if find_library("custatevec") is None and not imp_util.find_spec(
+        "cuquantum"
+    ):  # pragma: no cover
+        raise ImportError(
+            'cuQuantum libraries not found. Please check your "LD_LIBRARY_PATH" environment variable,'
```

Comment:
Should the warning mention `custatevec` instead and favor pip installing in a venv before setting `LD_LIBRARY_PATH`?

Reply:
Given how often this appears in the forum, I'd agree with the pip install error.

Reply:
Good idea!